packages/dev/parcel-optimizer-react-client/ReactClientOptimizer.js

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2023 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+const {Optimizer} = require('@parcel/plugin');
+const {blobToString} = require('@parcel/utils');
+
+module.exports = new Optimizer({
+  async optimize({bundle, contents, map}) {
+    if (!/@react-spectrum|react-aria-components/.test(bundle.target.distDir)) {
```

Comment:
need to worry about any of the components we have in react-aria? VisuallyHidden, FocusRing, FocusScope

Reply:
I was thinking we'd put it in our components but that if you were using those lower level APIs you'd probably have "use client" in your own component anyway. Maybe we'll get some feedback and we can always add it later.